### PR TITLE
Python deploy

### PR DESCRIPTION
**Pull request by janelletavares**
## Description of change

Fixes https://github.com/meroxa/turbine-project/issues/137

## Type of change

- [ ]  New feature
- [ ]  Bug fix
- [ ]  Refactor
- [ ]  Documentation

## How was this tested?

- [ ]  Unit Tests
- [ ]  Tested in staging
- [ ]  Tested in minikube

### CODE REVIEW

File: cmd/meroxa/root/apps/deploy.go [hunk 1]
```diff
@@ -352,13 +353,12 @@ func (d *Deploy) getPlatformImage(ctx context.Context, appPath string) (string,
...
 	buildInput := &meroxa.CreateBuildInput{SourceBlob: sourceBlob}
 
-	d.logger.StartSpinner("\t", " Building Meroxa Process image...")
-
 	build, err := d.client.CreateBuild(ctx, buildInput)
 	if err != nil {
 		d.logger.StopSpinnerWithStatus("\t", log.Failed)
 		return "", err
 	}
+	d.logger.StartSpinner("\t", fmt.Sprintf(" Building Meroxa Process image (%s)...", build.Uuid))
```
Comment (janelletavares) [inline cmd/meroxa/root/apps/deploy.go:361]: todo... undo. for debugging

File: cmd/meroxa/turbine_cli/python/deploy.go [hunk 2]
```diff
@@ -23,6 +37,26 @@ func NeedsToBuild(path string) (bool, error) {
...
+	cmd.Env = os.Environ()
+	cmd.Env = append(cmd.Env, fmt.Sprintf("MEROXA_ACCESS_TOKEN=%s", accessToken))
+
+	return turbinecli.RunCmdWithErrorDetection(ctx, cmd, l)
+}
 
-// TODO: Have a script to cleanup the temp directory used (right after source is uploaded)
+// CleanUpApp removes any temporary artifacts in the temp directory.
+func CleanUpApp(path string) (string, error) {
+	cmd := exec.Command("turbine-py", "cliclean", path)
```
Comment (janelletavares) [inline cmd/meroxa/turbine_cli/python/deploy.go:56]: @todo call this somewhere

File: cmd/meroxa/root/apps/deploy.go [hunk 3]
```diff
@@ -416,8 +422,7 @@ func (d *Deploy) buildApp(ctx context.Context) error {
 	case JavaScript:
 		d.tempPath, err = turbineJS.BuildApp(d.path)
 	case Python:
-		// TODO: @eric
-		// path only for zipping = turbinePy.BuildApp(ctx, d.logger, d.path) => newPath string
+		d.tempPath, err = turbinePY.BuildApp(d.path)
 		// Dockerfile will already exist
```
Comment (janelletavares) [inline cmd/meroxa/root/apps/deploy.go:426]:
```suggestion
		// Dockerfile will already exist
		d.tempPath, err = turbinePY.BuildApp(d.path)
```

**Review by ericcheatham — APPROVED** [review]
👍

**Pull request merged**